rest/api_test.go

```diff
@@ -217,29 +217,33 @@ func TestCORSOrigin(t *testing.T) {
 	reqHeaders := map[string]string{
 		"Origin": "http://example.com",
 	}
-	response := rt.SendRequestWithHeaders("GET", "/{{.keyspace}}/", "", reqHeaders)
+	response := rt.SendRequestWithHeaders("GET", "/{{.db}}/", "", reqHeaders)
+	RequireStatus(t, response, http.StatusUnauthorized)
```

Comment:
Were these tests previously returning these statuses, or does that change in this PR?

Reply:
Yes, using the same.